setup.py

```diff
@@ -6,20 +6,18 @@
 ROOT_DIR = os.path.dirname(__file__)
 SOURCE_DIR = os.path.join(ROOT_DIR)
 
-requirements = ["nose==1.3.0", "splinter>=0.4.9", "lettuce>=0.2.19"]
+requirements = ["nose", "splinter", "lettuce"]
```

Comment:
-1.  

Version-number specification is a very good thing, and prevents salad from randomly breaking if one of those packages introduces a backwards-incompatible release.